spec/lib/diaspora_federation/entities/comment_spec.rb

```diff
@@ -32,6 +32,8 @@ module DiasporaFederation
 
     it_behaves_like "a relayable Entity"
 
+    include_examples "#to_json output matches JSON schema"
```

Comment:
With `include_examples` we have a better full example name:
```
DiasporaFederation::Entities::Comment #to_json output matches JSON schema
```
while with `it_behaves_like` it gives:
```
DiasporaFederation::Entities::Comment behaves like #to_json output matches JSON schema #to_json output matches JSON schema
```
which can be changed to
```
DiasporaFederation::Entities::Comment behaves like #to_json output matches JSON schema
```
if we make `it` argument to be `""` at the shared_example definition, but we still have the "behaves like" words here that doesn't make any sense.

Since we have no technical difference in this case (the context of shared examples doesn't introduce anything, that could affect the context where we do include), I think we should pick the one which makes better name. However, if you insist, I can change it.